samcli/local/docker/lambda_image.py

```diff
@@ -287,3 +292,22 @@ def _generate_dockerfile(base_image, layers):
         for layer in layers:
             dockerfile_content = dockerfile_content + f"ADD {layer.name} {LambdaImage._LAYERS_DIR}\n"
         return dockerfile_content
+
+    def _remove_rapid_images(self, repo: str) -> None:
+        """
+        Remove all rapid images for given repo
+
+        Parameters
+        ----------
+        repo str
+            Repo for which rapid images will be removed
+        """
+        LOG.info("Removing rapid images for repo %s", repo)
+        try:
+            for image in self.docker_client.images.list(name=repo):
+                for tag in image.tags:
+                    if ":rapid-" in tag:
```

Comment:
not sure if this is enough or if I should do some regex like https://github.com/aws/aws-sam-cli/blob/a77d9e09e3739be6d802efa1394f2c8fe30ef13d/samcli/local/docker/manager.py#L200

Reply:
I guess we could maybe take this functionality out in its own function, in `samcli/local/docker/utils.py` I guess